src/module/combat/defense/PsychicDefense.svelte.js

```diff
@@ -61,16 +61,15 @@ export class PsychicDefense extends Defense {
     if (!this.newShield) {
       this.supernaturalShield = this.defender.system.combat.supernaturalShields[0];
     }
-
-    this.preventFatigue = this.defender.system.psychic.psychicSettings.fatigueResistance;
```

Comment:
Como en el ataque psíquico, si no usamos `this.preventFatigue` ya, debemos borrar la declaración del valor en la línea 30.